crates/diesel_models/src/payment_method.rs

```diff
@@ -838,3 +842,37 @@ impl From<&PaymentMethodNew> for PaymentMethod {
         }
     }
 }
+
+#[derive(Debug, Clone, serde::Serialize, serde::Deserialize)]
+pub struct PaymentsMandateReferenceRecord {
+    pub connector_mandate_id: String,
+    pub payment_method_type: Option<common_enums::PaymentMethodType>,
```

Comment:
```suggestion
    pub payment_method_sub_type: Option<common_enums::PaymentMethodType>,
```
In v2 the PaymentMethodType is referred as payment_method_sub_type and PaymentMethod is referred as payment_method_type

Reply:
We'd need to update the API models, diesel and domain models as well. I'll take this up in a separate PR.